### PR TITLE
Add meta tags  for social sharing

### PR DESCRIPTION
**Pull request by tuntisz**
https://trello.com/c/XqZ3HKXQ/74-add-meta-tags-for-social-media-link-previews

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,6 +11,19 @@
     <%= javascript_pack_tag 'application' %>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,700&display=swap" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <% if request.path.include?('providers/') %>
+      <%# TODO: The title below should include the provider's first name #%>
+      <meta property="og:title" content="Request for support from a provider -- Feed a Doc" />
```
Comment (tuntisz) [inline app/views/layouts/application.html.erb]: I could use some Rails expertise here. I'd like to get the id out of the `request.path`, find the provider with that id and display their name. If the provider is not found for some reason, it should probably safely fallback.

Comment (tuntisz) [reply app/views/layouts/application.html.erb]: Can I use AR in the view? Is there some existing mechanism for doing this sort of thing?

Comment (tuntisz) [reply app/views/layouts/application.html.erb]: Andrew helped me resolve this :raised_hands:

**Review by cantino — APPROVED** [review]

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -11,6 +11,23 @@
...
+        <meta property="og:title" content="Request for support from a provider -- Feed a Doc" />
+      <% end %>
+      <meta property="og:description" content="View and respond to this request for help from a COVID-19 care provider." />
+      <meta property="og:image" content="<%= image_url "/images/Facebook_Shared_Link-Provider_Request.jpg" %>" />
+      <meta property="og:url" content="<%= request.protocol + request.host + request.path %>" />
+    <% elsif request.path == '/'  %>
+      <meta property="og:title" content="Feed a Doc: Care providers need our help to fight COVID-19" />
+      <meta property="og:description" content="Volunteer to help a doctor, nurse, or medical worker on the front lines of the COVID-19 pandemic. Care providers: Post a request and share it with your friends!" />
+      <meta property="og:image" content="<%= image_url "/images/Facebook_Shared_Link_Default.jpg" %>" />
+      <meta property="og:url" content="<%= request.protocol + request.protocol + request.host %>" />
```
Comment (cantino) [inline app/views/layouts/application.html.erb:28]: You have `request.protocol` twice.

**Pull request merged**